tests/suites/test_suite_net.function

```diff
@@ -26,3 +70,63 @@ void context_init_free( int reinit )
     goto exit;
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_PLATFORM_IS_UNIXLIKE */
+void poll_beyond_fd_setsize( )
+{
+    /* Test that mbedtls_net_poll does not misbehave when given a file
+     * descriptor beyond FD_SETSIZE. This code is specific to platforms
```

Comment:
I find "beyond" a bit vague, perhaps "greater than or equal to" would be more precise, even if more verbose.